docs/3.0rc/deploy/infrastructure-concepts/work-pools.mdx

```diff
@@ -210,7 +210,7 @@ For example:
 ┡━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━┩
 │ barbeque   │ docker         │ 72c0a101-b3e2-4448-b5f8-a8c5184abd17 │ None              │
 │ k8s-pool   │ kubernetes     │ 7b6e3523-d35b-4882-84a7-7a107325bb3f │ None              │
-| my-pool    │ process        │ cd6ff9e8-bfd8-43be-9be3-69375f7a11cd │ None              │
+ │ my-pool    │ process        │ cd6ff9e8-bfd8-43be-9be3-69375f7a11cd │ None              │
```

Comment:
@discdiver just curious - why was this change needed?

Reply:
It looked like that pipe character was different than all the others (my syntax highlighter had it in different color), so I just copied one of the other pipes for consistency.

Reply:
ah i see what you mean, thanks